File: tinytest.py
#!/usr/bin/env python
import os, sys

def run_script():
        return os.system('./application.php ' + ' '.join(sys.argv[1:]))

if '-l' in sys.argv or '--loop' in sys.argv:
	while True:
		ret = run_script()
		if ret == 2:
			break
else:
	run_script()

File: tinytest.php
#!/usr/bin/php
<?php
require("argument.php");

$argv = $argv or array();
$argsStr = argsToString($argv);

function argsToString( $argv ) {
    $argvSlice = array_slice($argv, 1);
    return implode(" ", $argvSlice);
}

$loopArgument = new LoopArgument();
if($loopArgument->inArguments($argv)) {
    while(true) {
        runScript($argsStr);
    }
}
else {
    runScript($argsStr);
}

function runScript( $argsStr) {
    system("./application.php " . $argsStr);
}
?>
